tf_helpers/lax.py

```diff
@@ -0,0 +1,505 @@
+# Copyright 2020 The Google/TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+# from tensorflow.compiler.xla.python import xla_client
```

Comment:
Here and elsewhere, please remove unused commented code. If applicable, replace it with TODOs/text to be more specific regarding the purpose of the comment.